Source/Demos/Demo.Tweening/Game1.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using MonoGame.Extended;
+using MonoGame.Extended.Animations;
+using MonoGame.Extended.Animations.Easing;
+using MonoGame.Extended.Animations.Fluent;
+using MonoGame.Extended.Animations.Transformations;
+using MonoGame.Extended.Sprites;
+using MonoGame.Extended.ViewportAdapters;
+
+namespace Demo.Tweening
+{
+    public class Game1 : Game
+    {
+        // ReSharper disable once NotAccessedField.Local
+        private readonly GraphicsDeviceManager _graphicsDeviceManager;
+        private SpriteBatch _spriteBatch;
+        private Sprite _sprite;
+        private Camera2D _camera;
+        private Animator _animator;
```

Comment:
I'd rather call this something more specific. We've already got a `SpriteSheetAnimator`. What type of animator is this? A `TweeningAnimator` maybe?  Also, I noticed it derives from `GameComponent`. I'd like to talk about having more game components later but I think they should all follow a consistent naming convention. e.g. `TweeningAnimatorComponent` or something.